client/src/test/java/com/sinch/sdk/domains/voice/api/v1/adapters/VoiceServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package com.sinch.sdk.domains.voice.api.v1.adapters;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.sinch.sdk.core.http.HttpClient;
+import com.sinch.sdk.models.ApplicationCredentials;
+import com.sinch.sdk.models.VoiceContext;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+class VoiceServiceTest {
```

Comment:
Can you also add a test where there is a context, an application key and an application secret? Just to be sure it works when all the parameters are present :)

Reply:
Right done